src/methods/function.js

```diff
@@ -600,16 +601,19 @@ function InternalCall(realm: Realm, F: FunctionValue, thisArgument: Value, argsL
     invariant(realm.getRunningContext() === callerContext);
 
     for (let t2 of realm.tracers)
-      t2.afterCall(F, thisArgument, argsList, undefined, result);
+      t2.afterCall(F, thisArgument, argsList, undefined, (result: any));
```

Comment:
Let's see. joinAndRemoveNestedReturnCompletions  is typed to possibly return a PossiblyNormalCompletion. The afterCall function is not typed to receive that. Of course, that new type of result value cannot reach the afterCall call. @schandragit, does that make sense, is Flow getting improved to avoid this issue?

Herman, did you consider restructuring the code a bit to avoid the :any cast?